SmartCar/Arduino/sketch_main/sketch_main.ino

```diff
@@ -57,16 +79,49 @@ void setup() {
     pinMode(FLAME_SENSOR_PIN, INPUT);
     
     Serial.begin(9600);  
+
+  // Start the sensor
+    sensors.begin();    
+    if (!sensors.getAddress(thermometer, 0)) Serial.println("Unable to find address for Device 0");  
```

Comment:
Why are you printing to the serial? Doesn't that get passed to the android client and cause issues?